app/src/auth/utils/loginInterceptor.js

```diff
@@ -30,6 +30,12 @@ function shouldRedirect(error) {
         const url = new window.URL(error.config.url)
         const me = new window.URL(formatApiUrl('users', 'me'))
         const keys = new window.URL(formatApiUrl('users', 'me', 'keys'))
+        const changePassword = new window.URL(formatApiUrl('users', 'me', 'changePassword'))
+
+        // shouldn't redirect if current password is wrong when changing password
```

Comment:
This fixes an issue when you mistype the old password and the response comes back as 401 that caused a redirect.